pkgs/tools/filesystems/gcsfuse/default.nix

```diff
@@ -0,0 +1,23 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "gcsfuse-${version}";
+  version = "81281027";
+  rev = "81281027c0093e3f916a6e611a128ec5c3a12ece";
+
+  goPackagePath = "github.com/googlecloudplatform/gcsfuse";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/googlecloudplatform/gcsfuse";
+    sha256 = "1lj9czippsgkhr8y3r7vwxgc8i952v76v1shdv10p43gsxwyyi9a";
+  };
+
+  # Fully vendored, apparently.
+  goDeps = null;
```

Comment:
To be very nit-picky can you remove "apparently" isn't not necessary.

Also the current practice appears to be just not providing this field. I would prefer the constancy..